### PR TITLE
C#: Add precision tags to `UnsafeDeserialization[UntrustedInput].ql`

### PR DESCRIPTION
**Pull request by hvitved**
[Query run on LGTM.com](https://lgtm.com/query/2566465574766208746/) shows no results on my set of sample projects for `UnsafeDeserializationUntrustedInput.ql`, and the query `UnsafeDeserialization.ql` is disabled by default.

@jf205 : In addition to the change not change, could you also take a look at the qhelp for these two queries (not actually modified on this PR)? Thanks.

### CODE REVIEW

File: change-notes/1.23/analysis-csharp.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ The following changes in version 1.23 affect C# analysis in all applications.
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
+| Deserialization of untrusted data (`cs/unsafe-deserialization-untrusted-input`) | security | Finds flow of untrusted input to calls to unsafe deserializers. |
 | Unsafe year argument for 'DateTime' constructor (`cs/unsafe-year-construction`) | reliability, date-time | Finds incorrect manipulation of `DateTime` values, which could lead to invalid dates. |
+| Unsafe deserializer (`cs/unsafe-deserialization`) | security | Finds calls to unsafe deserializers. |
```
Comment (jf205) [inline change-notes/1.23/analysis-csharp.md]: external/cwe/cwe-502 tag?

**Comment by hvitved** [comment]
> * Both `UnsafeDeserializationUntrustedInput.qhelp` and `UnsafeDeserialization.qhelp` have the same text for the `overview` section---is that intended?

Yes, it is intended.

File: csharp/ql/src/Security Features/CWE-502/UnsafeDeserialization.ql [hunk 2]
```diff
@@ -5,16 +5,13 @@
  * @kind problem
  * @id cs/unsafe-deserialization
  * @problem.severity warning
+ * @precision low
```
Comment (calumgrant) [inline csharp/ql/src/Security Features/CWE-502/UnsafeDeserialization.ql]: Where did this value come from? I think medium/warning might be sufficient in order to get the query run but excluded.

Comment (hvitved) [reply csharp/ql/src/Security Features/CWE-502/UnsafeDeserialization.ql]: I am not sure we even want to run it; It always produces a superset of the results produced by the more ambitious `UnsafeDeserializationUntrustedInput.ql` query.

**Pull request merged**